scripts/albs-1147.py

```diff
@@ -0,0 +1,181 @@
+import difflib
+import argparse
+import re
+import asyncio
+import datetime
+import logging
+import os
+import sys
+
+from contextlib import asynccontextmanager
+
+from sqlalchemy import select, or_, not_
+
+from alws.dependencies import get_pulp_db, get_db
+
+sys.path.append(os.path.dirname(os.path.dirname(__file__)))
+
+from alws.models import ErrataRecord
+from alws.pulp_models import UpdateRecord
+
+
+logging.basicConfig(
+    format="%(message)s",
+    level=logging.INFO,
+    datefmt="%m-%d %H:%M:%S",
+    handlers=[
+        logging.StreamHandler(),
+        logging.FileHandler("albs-1147.log"),
+    ],
+)
+log = logging.getLogger()
+CONTEXT_SIZE = 25
+
+
+def debrand_description_and_title(original_string: str) -> str:
+    branding_patterns = (
+        (r'RHEL', 'AlmaLinux'),
+        (r'\[rhel', '[almalinux'),
+        (r'\(rhel', '(almalinux'),
+        (r'connect to rhel server', 'connect to almalinux server'),
+        (r'kvm-rhel8.3', 'kvm-almalinux8.3'),
```

Comment:
This is the only one I'm not sure about. Did Andrew review them?

Reply:
Waiting on Andrew. This is my decision only.